test/test_linalg.py

```diff
@@ -8029,38 +8029,56 @@ def fn(torchfn, *args):
         self.assertRaises(RuntimeError, lambda: torch.lstsq(torch.randn(0, 0), torch.randn(0, 0)))
         self.assertRaises(RuntimeError, lambda: torch.lstsq(torch.randn(0,), torch.randn(0, 0)))
 
+    @dtypes(*floating_types())
```

Comment:
We can probably just run this in tf32 still unless there's a particular reason this would like to test double, too?

Reply:
yes, there is no particular reason. I switched to testing only `torch.float32`.

Reply:
yes, there is no particular reason. I switched to testing only torch.float32 in 33345f7cdf